### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ColumbusDeveloper**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,17 +1,164 @@
...
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
```
Comment (YegorKorenkov) [inline src/index.html]: also, please add method
image

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,164 @@
...
+        <fieldset>
+          <legend>Personal information</legend>
+          <label>
+            Surname:
+            <input
+             name="surname"
```
Comment (YegorKorenkov) [inline src/index.html]: Must be two spaces offset(but now 1)
image

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,164 @@
...
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="label">
+            <span>Do you love cats? </span>
+            <input
+              id="cats_1"
+              type="radio"
+              name="cats"
+            >
+            <label for="cats_1" class="cats_1">Yes</label>
```
Comment (YegorKorenkov) [inline src/index.html]: Please use camelCase for attributes 
image

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,164 @@
...
+            Surname:
+            <input
+             name="surname"
+             type="text"
+             autocomplete="off"
+            >
+          </label>
+          <label>
+            Name:
+            <input type="text"
```
Comment (YegorKorenkov) [inline src/index.html]: Please format your code, check for all cases!
image

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,164 @@
...
+            <div class="select-shell">
+              <select
+                name="brand"
+                class="select-brand"
+                multiple
+              >
+                <option value="BMV">BMW</option>
+                <option value="Audi">Audi</option>
+                <option value="Lada">Lada</option>
+                <option
```
Comment (YegorKorenkov) [inline src/index.html]: Please remove this option, it redundant

File: src/index.html [hunk 6]
```diff
@@ -1,17 +1,164 @@
...
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
```
Comment (YegorKorenkov) [inline src/index.html]: according to the conditions of the task, you do not need this container

File: src/style.css [hunk 7]
```diff
@@ -1 +1,80 @@
-/* styles go here */
+/* stylelint-disable block-opening-brace-space-before */
```
Comment (YegorKorenkov) [inline src/style.css:1]: why you need all of this styles? please remove all and try to add only two classes for vertical distance
image

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 7 above)
Comment (ColumbusDeveloper) [reply src/style.css:1]: just wanted to make the most of the possibilities of css.
I agree, the simpler the better.

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**